securesystemslib/__init__.py

```diff
@@ -9,3 +9,10 @@
 logger = logging.getLogger(__name__)
 logger.setLevel(logging.WARNING)
 logger.addHandler(logging.StreamHandler())
+
+
+# Global constants
+# TODO: Replace hard-coded key types with these constants (and add more)
```

Comment:
Would these constants be better suited to the keys module? Is it worth documenting how we're choosing where to put constants?

Reply:
IMO `__init__` is a good default place for global constants. If we want to module-scope them, we should coordinate with #270.